File: app/src/main/java/com/katkov/lolachievements/application/base/ErrorView.java
package com.katkov.lolachievements.application.base;

import com.arellomobile.mvp.viewstate.strategy.OneExecutionStateStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;

public interface ErrorView {

    @StateStrategyType(OneExecutionStateStrategy.class)
    void showError(Error error);
}

File: app/src/main/java/com/katkov/lolachievements/application/base/ErrorView.kt
package com.katkov.lolachievements.application.base

import com.arellomobile.mvp.viewstate.strategy.OneExecutionStateStrategy
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType

interface ErrorView {

    @StateStrategyType(OneExecutionStateStrategy::class)
    fun showError(error: Error)
}
